File: JavaScript/LinkedList/Node/Node.js
/**
 * LinkedList Definition
 */

class Node {
  constructor(data) {
    this.data = data;
    this.next = null;
  }

  /**
   * Print a LinkedList
   * @param {*} head: Head Node of LL
   */
  static printList(head) {
    let currentNode = head;
    while (currentNode) {
      console.log(currentNode.data);
      currentNode = currentNode.next;
    }
  }
}

export default Node;

File: python/LinkedList/Node/Node.py
"""
Node Definition Class.
"""

class Node:
    def __init__(self, data: int ):
        self.data = data
        self.next = None


    @staticmethod
    def print_linked_list(head: 'Node') -> None:
        """
        Print a LinkedList
        :param head: Head Node
        :return:
        """

        current_node = head

        while current_node is not None:
            print(current_node.data, end=" ==> ")

            if current_node.next is None:
                print(current_node.data)
            current_node = current_node.next


